nilearn/decoding/decoder.py

```diff
@@ -875,7 +879,7 @@ def _predict_dummy(self, n_samples):
 
 @fill_doc
 class Decoder(_BaseDecoder):
-    """A wrapper for popular classification strategies in neuroimaging.
+    r"""A wrapper for popular classification strategies in neuroimaging.
```

Comment:
Why use raw string here?

Reply:
was getting a:
```
nilearn/decoding/decoder.py:882:1: D301 Use r""" if any backslashes in a docstring
```

so I obediently complied. 🙈 

Now pushed this that should be a better fix:
https://github.com/nilearn/nilearn/pull/3534/commits/6b253036a97646bfdbab2d87705956de3d4bf929

Reply:
Oh I see haha. Ok better now thanks!